### PR TITLE
configure dependabot

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gitsubmodule"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
```
Comment (squaremo) [inline .github/dependabot.yml:17]: Could/Should this be more specific?

Comment (dholbach) [reply .github/dependabot.yml:17]: I stole from https://github.com/weaveworks/ignite/blob/master/.github/dependabot.yml and the example under https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem

Comment (squaremo) [reply .github/dependabot.yml:17]: Got it -- https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#directory advises:

> For GitHub Actions, set the directory to / to check for workflow files in .github/workflows.

**Review by squaremo — APPROVED** [review]
Let's give it a whirl!

**Pull request merged**